src/plugins/data_table/server_side_data_table.js

```diff
@@ -70,13 +70,49 @@ var ServerSideDataTable = (function() {
 
     // dataTables callback to allow addition of params to the ajax request
     _addServerParams : function(aoData) {
+      if (this.simpleParams) {
+        var sortBy, sortDir, limit, start, requestId;
+
+        var indexOfSortedColumn = this._getDataTableParam(aoData, "iSortCol_0");
+
+        if (indexOfSortedColumn) {
+          sortBy = this._columnManager.columnAttrs()[indexOfSortedColumn];
+          sortDir = this._getDataTableParam(aoData, "sSortDir_0");
+        }
+
+        limit = this._getDataTableParam(aoData, "iDisplayLength");
+        start = this._getDataTableParam(aoData, "iDisplayStart");
+        requestId = this._getDataTableParam(aoData, "sEcho");
+
+        // clear out existing array (but keeping reference to existing object)
+        aoData.splice(0, aoData.length);
+
+        aoData.push({ name: "sort_by",    value: sortBy });
+        aoData.push({ name: "sort_dir",   value: sortDir });
+        aoData.push({ name: "limit",      value: limit });
+        aoData.push({ name: "start",      value: start });
+        aoData.push({ name: "request_id", value: requestId });
```

Comment:
Is request_id used somewhere else? Or is it meant as like a memo field on a check?